manifests/application.pp

```diff
@@ -22,6 +22,8 @@
   include ::certs::apache
   include ::certs::foreman
   include ::certs::pulp_client
+  include ::certs::qpid
```

Comment:
Shouldn't qpid_client be sufficient? Or can we make it sufficient?

Reply:
Well, `certs::qpid_client` just rolls out certificates for the qpid-config tool. `certs::qpid` rolls out the whole nss certstore. We need the latter for the app.
`::katello::qpid_client` does include `include ::certs::qpid`, so this is here just for completeness' sake.

Reply:
If we're not using it in this class then I'd consider it an implementation detail of ```qpid_client```, but it might even be better to just include ```qpid::client``` in this class rather than split it off if there's just 1 use of it.

Reply:
I left this in now, as it's required for the notify of foreman-tasks.